src/bodywork/k8s/workflow_jobs.py

```diff
@@ -97,6 +103,18 @@ def configure_workflow_job(
     return job
 
 
+def _create_job_name(project_repo_url: str, project_repo_branch: str) -> str:
+    """Create unique job name.
+
+    :param project_repo_url: The URL for the Bodywork project Git
+        repository.
+    :param project_repo_branch: The Bodywork project Git repository
+        branch being used.
+    """
+    repo_name = os.path.splitext(os.path.basename(project_repo_url))[0]
```

Comment:
Might be easier with a regex (re module)?

Reply:
I find this a lot more readable than the regex solution.

Reply:
I've just tested this...

... and I agree 🙂